source/index.js

```diff
@@ -24,24 +25,34 @@ const npm = require('./npm/util');
 const releaseTaskHelper = require('./release-task-helper');
 const util = require('./util');
 const git = require('./git-util');
+const {of} = require('rxjs');
 
 const exec = (cmd, args) => {
 	// Use `Observable` support if merged https://github.com/sindresorhus/execa/pull/26
 	const cp = execa(cmd, args);
 
+	if (!cp.stdout || !cp.stdout.pipe) {
```

Comment:
I kept getting errors when running the `ava` unit tests about `cp.stdout.pipe` and `cp.stderr.pipe` not existing.

Looking at the test impl using `sinon.sub`, I don't understand how this code was stubbing and working correctly previously, so hopefully someone can tell me what I'm missing 😄